tests/Mantis/Helper/ArrayTransposeTest.php

```diff
@@ -0,0 +1,132 @@
+<?php
+
+declare(strict_types=1);
```

Comment:
Move below header comments + coding guidelines

Reply:
you mean the array style 

`[ [ 'a', 'b', ] ]` to `[['a', 'b']]`?

Reply:
I am fiddeling with the ecs and your sniffs, 

> Arrays elements should be referenced without spaces, e.g. $arr['index'] not $arr[ 'index' ]

Hopefully such will fixed with the "ArraySyntax" fixer in the future.

Reply:
array style fixed.

Reply:
> you mean the array style

Actually I meant the spacing around parentheses and equal sign `declare( strict_types = 1 );`

But you did right with the arrays too